### DISCUSSION TITLE
v3.2.0

### DISCUSSION THREAD

**Discussion by LinuxSuRen** [kubesphere/ks-devops#338] [Announcements]
### New Feature：

- Feat: Hide the branch column if is not a multi-branch pipeline ([#2379](https://github.com/kubesphere/console/pull/2379))
- Feat: Support load Jenkins configuration from ConfigMap automatically ([#75](https://github.com/kubesphere/ks-devops/pull/75))
- Feat: Support trigger Pipeline run by manipulate CRDs instead of calling Jenkins API ([#41](https://github.com/kubesphere/ks-devops/issues/41))
- Feat: Add containerd support for Pipeline ([#171](https://github.com/kubesphere/ks-devops/pull/171))
- Feat: Put Jenkins Job metadata into the annotations of Pipeline ([#254](https://github.com/kubesphere/ks-devops/issues/254))

### Bug Fix：

- Fix: Failed to create or update credential while the length of content is too long ([#123](https://github.com/kubesphere/ks-devops/pull/123))
- Fix: ks-apiserver crashed while viewing parallel Pipeline activity detail ([#93](https://github.com/kubesphere/ks-devops/pull/93))
- Fix: When branch name is invalid, report a error message #328 @mangoGoForward

### Dependency Upgrades

- Upgrade to configuration-as-code to 1.53 ([#42](https://github.com/kubesphere/ks-jenkins/pull/42))

## API

- Deprecate router API version v1alpha2. ([#4193](https://github.com/kubesphere/kubesphere/pull/4193))
- Change the pipeline api from v2 to v3 ([#2323](https://github.com/kubesphere/console/pull/2323))
- Change the secret verify api ([#2368](https://github.com/kubesphere/console/pull/2368))

There are 46 GitHub stars and 27 forks in this release.

Thanks again to @AbdelouahabMbarki, @JohnNiang, @LinuxSuRen, @cndoit18, @flamywhale, @fossabot,  @mangoGoForward, @seanly, @shihaoH, @yJunS and @yuezhuangshi! 🎉

<hr /><em>This discussion was created from the release <a href='https://github.com/kubesphere/ks-devops/releases/tag/v3.2.0'>v3.2.0</a>.</em>

**Comment by Dishone** [comment]
Hi @LinuxSuRen.I have upgraded kubersphere 3.2.0. I did not find a .Net 5 image in kubesphere devops.Are there any difficulties in the process?

**Reply by JohnNiang** [reply]
Hi @Dishone , please see the failed action: https://github.com/kubesphere/devops-agent/actions/runs/1398472523

**Reply by JohnNiang** [reply]
And please follow up https://github.com/kubesphere-sigs/ks-devops-helm-chart/issues/28 for .Net 5 agent.

**Reply by Dishone** [reply]
> And please follow up [kubesphere-sigs/ks-devops-helm-chart#28](https://github.com/kubesphere-sigs/ks-devops-helm-chart/issues/28) for .Net 5 agent.

Okay, I'm trying to locate this error
